src/Runner.Sdk/Util/VssUtil.cs

```diff
@@ -85,6 +85,35 @@ public static void InitializeVssClientSettings(List<ProductInfoHeaderValue> addi
             VssCredentials credentials,
             IEnumerable<DelegatingHandler> additionalDelegatingHandler = null,
             TimeSpan? timeout = null)
+        {
```

Comment:
is these change just moving content around?  create the new `GetHttpRequestSettings` for re-use

Reply:
Yup, line 116 is the new method and it's reused in the `ResultsServer`.